### PR TITLE
Remove sorting arrows on mobile

### PR DESCRIPTION
**Pull request by FrederikBolding**

### CODE REVIEW

**Comment by FrederikBolding** [comment]
[ch7453]

File: src/components/AccountList.tsx [hunk 1]
```diff
@@ -339,6 +339,9 @@ const getSortingFunction = (sortKey: ISortTypes): TSortFunction => {
   }
 };
 
+export const screenIsMobileSized = (): boolean =>
+  window.matchMedia(`(max-width: ${breakpointToNumber(BREAK_POINTS.SCREEN_XS)}px)`).matches;
+
```
Comment (GuillaumeRx) [inline src/components/AccountList.tsx:344]: we have `isMobile` from `useScreenSize` for this

File: src/features/Settings/components/AddressBook.tsx [hunk 2]
```diff
@@ -125,6 +125,9 @@ const getSortingFunction = (sortKey: ISortTypes): TSortFunction => {
   }
 };
 
+export const screenIsMobileSized = (): boolean =>
+  window.matchMedia(`(max-width: ${breakpointToNumber(BREAK_POINTS.SCREEN_XS)}px)`).matches;
+
```
Comment (GuillaumeRx) [inline src/features/Settings/components/AddressBook.tsx:130]: Same here

**Review by GuillaumeRx — APPROVED** [review]

**Review by GuillaumeRx — CHANGES_REQUESTED** [review]
Missclicked on the last review

File: src/features/Settings/components/AddressBook.tsx (hunk 2 above)
Comment (FrederikBolding) [reply src/features/Settings/components/AddressBook.tsx:130]: It has to match the breakpoint for the table. I have simplified.

File: src/components/AccountList.tsx (hunk 1 above)
Comment (FrederikBolding) [reply src/components/AccountList.tsx:344]: It has to match the breakpoint for the table. I have simplified.

**Review by GuillaumeRx — APPROVED** [review]

**Pull request merged**